js/everything.js

```diff
@@ -22,9 +22,15 @@ function onYouTubeIframeAPIReady() {/* jshint ignore:line */
                 //console.log("State changed to " + event.data);
                 var playerState = event.data;
 
-                 if (playerState === YT.PlayerState.ENDED) {
+                if (playerState === YT.PlayerState.ENDED) {
```

Comment:
You can make another commit, hop in the gitter chat if you need anything :smile: